### DISCUSSION TITLE
memos-api/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#104] [Giscus]
# memos-api/

Memos 简介系列 《Memos 简介》 《Memos 手动导入数据》 《Memos API 调用渲染页面》 《Memos API 公告样式滚动效果》 《Memos API 获取总条数》

https://www.eallion.com/memos-api/

**Comment by fengyiyi** [comment]
不知道哪里出问题了，我的https://m.1v1.design/api/memo?creatorId=1&rowStatus=NORMAL&limit=10 打开是404

**Reply by eallion** [reply]
0.19.x 之后的版本，可能会有这个问题。
